src/lua/compat.lua

```diff
@@ -4,19 +4,40 @@
 local NEW = true
 local OLD = false
 
-local internals = require('compat_internals') -- luacheck: no unused
+local internals = require('compat_internals')
 
 local options_format = {
     default  = 'string',
     brief    = 'string',
     obsolete = 'boolean',
 }
 
+local JSON_ESCAPE_BRIEF = [[
+Whether to escape the forward slash symbol '/' using a backslash in a json.encode()
+result. The old and the new behavior produce a result, which is compatible
+with the JSON specification. However most of other JSON encoders don't escape
+the forward slash, so the new behavior is considered more safe.
+https://github.com/tarantool/tarantool/wiki/compat%3Ajson_escape_forward_slash
```

Comment:
Nit: I would place the URL in its own paragraph. IOW, I would add an empty line before it.

Reply:
Fixed.